LayoutTests/imported/w3c/web-platform-tests/fullscreen/api/fullscreen-reordering.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<title>Re-requesting fullscreen doesn't fail but doesn't change order</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="../../html/semantics/popovers/resources/popover-utils.js"></script>
+
+<div class="elements">
+  <div id="A">Element A</div>
+  <div id="B">Element B</div>
+</div>
+
+<style>
+  .elements>div {
+    width:200px;
+    height:200px;
+  }
+  #A { background: blue; }
+  #B { background: green; }
+</style>
+
+<script>
+promise_test(async (t) => {
+  t.add_cleanup(async () => {
+    while (document.fullscreenElement)
+      await document.exitFullscreen();
+  });
+  document.onfullscreenerror = () => assert_unreached('fullscreenerror should not happen');
+  const A = document.getElementById('A');
+  const B = document.getElementById('B');
+  assert_true(!isTopLayer(A) && !isTopLayer(B));
+  await blessTopLayer(document.body);
+  await A.requestFullscreen();
+  assert_equals(document.fullscreenElement,A,'first A request');
```

Comment:
nit, white space between the comas... might be good to tidy this consistently

Reply:
this is a test imported from upstream, I can clean it up upstream later